samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml

```diff
@@ -0,0 +1,163 @@
+metadata:
+  version: "1"
+  dependencies:
+    apps:
+      - id: dynatrace.automations
+        version: ^1.728.0
+      - id: dynatrace.jira
+        version: ^4.1.1
+  inputs:
+    - type: connection
+      schema: app:dynatrace.jira:connection
+      targets:
+        - tasks.create_task_in_jira.connectionId
+workflow:
+  title: "[Sample] Jira tickets for critical vulnerabilities"
+  tasks:
+    get_vulnerabilites:
+      name: get_vulnerabilites
+      description: Query for new vulnerability findings in the last 24 hours.
+      action: dynatrace.automations:execute-dql-query
+      active: true
+      input:
+        query: >-
+          fetch events, from: now() - 24h
+
+          // data access
+
+          | filter dt.system.bucket == "default_security_custom_events"
+               AND event.type == "VULNERABILITY_FINDING_EVENT"
+               AND isNotNull(vulnerable_component.name)
+          | dedup {affected_entity.id, vulnerability.external_id,
+          vulnerable_component.name}, sort: {timestamp desc}
+
+          // aggregation and custom filtering
+
+          | filter vulnerability.risk.level=="CRITICAL"
+
+          | summarize {
+                vulnerable_components_count = count(),
+                vulnerability_finding_events = collectArray(
+                  record(
+                    affected_entity.id = affected_entity.id,
+                    affected_entity.name = affected_entity.name,
+                    event.provider_product = event.provider_product,
+                    vulnerable_component.version = vulnerable_component.version,
+                    vulnerable_component.short_name = vulnerable_component.short_name,
+                    vulnerability.risk.level = vulnerability.risk.level,
+                    vulnerability.title = vulnerability.title,
+                    vulnerability.external_id = vulnerability.external_id,
+                    vulnerability.description = vulnerability.description,
+                    scan_time = timestamp
+                  )
+                )
+              }, by:{ vulnerability.external_id}
+          | filterOut iAny(vulnerability_finding_events[][scan_time] < now() -
+          24h)
+
+          | sort vulnerability_finding_events[][scan_time] desc
+
+          | limit 1
+      customSampleResult: {}
+      position:
+        x: 0
+        y: 1
+      predecessors: []
+      conditions:
+        states: {}
+        custom: ""
+    create_task_in_jira:
+      name: create_task_in_jira
+      description: Create a Jira ticket for the vulnerability findings.
+      action: dynatrace.jira:jira-create-issue
+      active: true
+      input:
+        labels: []
+        taskId: "{{ task().id }}"
+        project:
+          id: "19691"
```

Comment:
The project id is specific to the Jira instance.

Reply:
Thank you,  removed the projectId